packages/dx-grid-core/src/plugins/table-group-row/helpers.ts

```diff
@@ -86,3 +87,20 @@ export const calculateGroupCellIndent: PureComputed<[TableColumn, Grouping[], nu
 ) => (
   indentWidth * getGroupIndexByColumn(grouping, tableColumn)
 );
+
+export const sortAndSpliceColumns: PureComputed<[TableColumn[], number]> = (
+  tableColumns, firstVisibleColumn,
+) => {
```

Comment:
`firstVisibleColumn` -> `firstVisibleColumnIndex` or something else... because it is a number type ?